<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExampleConfiguration</class>
 <widget class="QDialog" name="ExampleConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Example settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Button Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="iconCB">
        <property name="text">
         <string>Show icon:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="iconLE"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="textCB">
        <property name="text">
         <string>Show text:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="textLE"/>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Keyboard Shortcut</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Click the button to record shortcut:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ShortcutSelector" name="shortcutEd">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>163</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutSelector</class>
   <extends>QToolButton</extends>
   <header location="global">LXQtGlobalKeysUi/ShortcutSelector</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>iconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
